src/frontend/src/components/Post/telescope-post-content.css

```diff
@@ -41,6 +41,12 @@
   vertical-align: center;
 }
 
+.telescope-post-content img[src*='https://medium.com/_/stat']
+{
```

Comment:
Move this up to 44 with the selector please.

Reply:
sorry, auto save on format changed in my VS Code for some reason